mgmt_cli_test.py

```diff
@@ -459,6 +470,55 @@ def get_email_data(self):
 
         return email_data
 
+    def test_backup_and_restore_only_data(self):
+        self.run_prepare_write_cmd()
+        manager_tool = mgmt.get_scylla_manager_tool(manager_node=self.monitors.nodes[0])
+        mgr_cluster = manager_tool.get_cluster(cluster_name=self.CLUSTER_NAME) \
+            or manager_tool.add_cluster(name=self.CLUSTER_NAME, db_cluster=self.db_cluster,
+                                        auth_token=self.monitors.mgmt_auth_token)
+        backup_task = mgr_cluster.create_backup_task(location_list=self.locations)
+        backup_task_status = backup_task.wait_and_get_final_status(timeout=110000)
+        assert backup_task_status == TaskStatus.DONE, \
+            f"Backup task ended in {backup_task_status} instead of {TaskStatus.DONE}"
+        self.db_cluster.nodes[0].run_cqlsh('TRUNCATE keyspace1.standard1')
+        self.restore_data_with_task(mgr_cluster=mgr_cluster, backup_task=backup_task, timeout=110000)
+        self.run_verification_read_stress()
+
+    @staticmethod
+    def _get_persistent_snapshots():
+        with open("defaults/manager_persistent_snapshots.yaml", encoding="utf-8") as mgmt_snapshot_yaml:
+            persistent_manager_snapshots_dict = yaml.safe_load(mgmt_snapshot_yaml)
+        return persistent_manager_snapshots_dict
+
+    def test_restore_multiple_keyspaces(self):
```

Comment:
I don't understand the purpose of the test.
From its name it may be sound that the purpose is to test a restore of multiple keyspaces.
In that case, I would expect the test to (backup?) multiple keyspace (> 10, 50?, 100?) and try to restore them.
(It could also restore the same one to multiple new keyspaces)

However, from the code it looks like it just restore 2 keyspaces that were pre-backup and stored in AWS.
And from what I know, you do it mostly because you needed this as a way to restore to a clean cluster including the schema).